scala/src/main/scala/smile/clustering/distances/Binary/MeanMahanttan.scala

```diff
@@ -2,9 +2,9 @@ package smile.clustering.distances
 
 import scala.math.pow
 
-class MeanMahanttan extends Binary_Distance
-{
-	override def distance(vector1: Array[Int], vector2: Array[Int], specialArg: String = "") : Double =
+class MeanMahanttan extends BinaryDistance {
```

Comment:
What's difference from MahanttanDistance? If different, can we leverage the `MahanttanDistance` class in Java in the computation to simplify the code?